test/errors/managedRecord-detailed.good

```diff
@@ -0,0 +1,15 @@
+─── error in managedRecord.chpl:4 [MemManagementNonClass] ───
+  Cannot use memory management strategy 'shared' with record 'MyRecord'.
```

Comment:
Is "strategy" the term we use in the spec?  (in this context, it sounds almost more like a hint or pragma than a first-class language concept; but if that's what we're using in defining it, we should be consistent here).

Reply:
It is a spec term: https://chapel-lang.org/docs/language/spec/classes.html#class-types